x/stakeibc/module_ibc.go

```diff
@@ -211,7 +186,40 @@ func (im IBCModule) NegotiateAppVersion(
 	return proposedVersion, nil
 }
 
+func (im IBCModule) UnmarshalAck(ctx sdk.Context, acknowledgement []byte) (*channeltypes.Acknowledgement_Result, error) {
+	ack := channeltypes.Acknowledgement_Result{}
+	eventType := "callback"
```

Comment:
is this function unique to icacallbacks? I assume it could work with regular ibc acks - could it make sense to pass an eventType as an arg?

Reply:
pulled this event out of `UnmarshalAck`